### DISCUSSION TITLE
Utilities is disapeared since the last update

### DISCUSSION THREAD

**Discussion by AtomikRemy** [AzuraCast/AzuraCast#6271] [Q&A]
... and it's more difficult to create sftp users, for example.
Thks...

**Comment by BusterNeece** [comment]
@AtomikRemy The menu items rearranged several updates ago, almost a year ago at this point if I recall. SFTP users are under "Media" now.

**Reply by AtomikRemy** [reply]
Yes, thanks, I finaly found. But so I only can create sftp user for each radio. I'd like to create one superuser sftp to place my backups...
